ru/mail/chrome/messenger/messengercompose/pbMacroSearchOverlay.dtd

```diff
@@ -1,9 +1,9 @@
 <!ENTITY pbMacroSearchPanel.width "600">
 
 <!ENTITY insertMacro.key "l">
-<!ENTITY insertMacro.modifiers "accel">
+<!ENTITY insertMacro.modifiers "����������">
```

Comment:
@yshchebet This line does not look correct.  It has garbage characters.  +<!ENTITY insertMacro.modifiers "����������">